vendor/github.com/grpc-ecosystem/grpc-gateway/internal/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
Now sure why the OWNER and BUILD files are added, afaik they should be filtered out by some mean. Could you remove those from the vendor dir ?

Reply:
these files were automatically created when building the sources with hack/build.sh

this time I forgot to exclude when doing the git commit

Reply:
have removed the files form git commit